palindr.js

```diff
@@ -0,0 +1,16 @@
+s = "adfghjkjhgfdarrrrrfffffvhnjbjhnjnghjmkdghjgdhj"
+function IskatelPolyndromusov(str) {
+    var polydeo = "";
+    for (var i = 0; i < str.length; i++) {
+        for (var u = i + 1; u <= str.length; u++) {
+            var subString = str.slice(i,u);
+            if (subString === subString.split('').reverse().join('') && subString.length > polydeo.length){
```

Comment:
We cannot use` reverse and split` :) 
@Chapalah336 I said this many times! You are not listening